src/server/rest/v1/service/AuthorizationService.ts

```diff
@@ -313,39 +261,42 @@ export default class AuthorizationService {
       authorized: userToken.role === UserRole.ADMIN,
     };
     // Filter projected fields
-    authorizationFilters.projectFields = AuthorizationService.filterProjectFields(
-      authorizationFilters.projectFields, filteredRequest.ProjectFields);
+    authorizationFilters.projectFields = AuthorizationService.filterProjectFields(authorizationFilters.projectFields,
+      filteredRequest.ProjectFields);
     // Get authorization filters from users
-    const usersAuthorizationFilters = await AuthorizationService.checkAndGetUsersAuthorizationFilters(
-      tenant, userToken, filteredRequest);
+    const usersAuthorizationFilters = await AuthorizationService.checkAndGetUsersAuthorizationFilters(tenant,
+      userToken, filteredRequest);
     // Override
     authorizationFilters.authorized = usersAuthorizationFilters.authorized;
     authorizationFilters.filters = usersAuthorizationFilters.filters;
     return authorizationFilters;
   }
 
-  public static async addUsersAuthorizations(tenant: Tenant, userToken: UserToken, users: User[], authorizationFilter: AuthorizationFilter): Promise<void> {
+  public static async addUsersAuthorizations(tenant: Tenant, userToken: UserToken, users: User[],
+      authorizationFilter: AuthorizationFilter): Promise<void> {
     // Enrich
     for (const user of users) {
       await AuthorizationService.addUserAuthorizations(tenant, userToken, user, authorizationFilter);
     }
   }
 
-  public static async addUserAuthorizations(tenant: Tenant, userToken: UserToken, user: User, authorizationFilter: AuthorizationFilter): Promise<void> {
+  public static async addUserAuthorizations(tenant: Tenant, userToken: UserToken, user: User,
+      authorizationFilter: AuthorizationFilter): Promise<void> {
     // Enrich
     if (!user.issuer) {
       user.canRead = true;
       user.canUpdate = false;
       user.canDelete = false;
     } else {
-      user.canRead = await AuthorizationService.canPerformAuthorizationAction(tenant, userToken, Entity.USER, Action.READ, authorizationFilter);
+      user.canRead = await AuthorizationService.canPerformAuthorizationAction(tenant, userToken, Entity.USER, Action.READ,
+        authorizationFilter);
       user.canUpdate = await Authorizations.canUpdateUser(userToken, user.id);
       user.canDelete = await Authorizations.canDeleteUser(userToken, user.id);
     }
   }
 
   public static async checkAndGetUsersAuthorizationFilters(tenant: Tenant, userToken: UserToken,
-      filteredRequest: HttpUsersRequest): Promise<AuthorizationFilter> {
+      filteredRequest?: Partial<HttpUsersRequest>): Promise<AuthorizationFilter> {
```

Comment:
I think that argument should not be optional, the whole purpose of that method is to check the request.